backend/corpora/dataset_processing/process.py

```diff
@@ -46,6 +64,23 @@ def fix_dropbox_url(url):
     return pr.geturl()
 
 
+def update_db(metadata=None, processing_status=None):
```

Comment:
you'll need to wrap this in @db_session to make sure the changes are committed or rolled back.

Reply:
https://github.com/chanzuckerberg/corpora-data-portal/blob/7b3255bfe5c075de4da6c8ea65defa5111ae67b0/backend/corpora/common/utils/db_utils.py#L71